en/02_Developer_Guides/00_Model/01_Data_Model_and_ORM.md

```diff
@@ -698,16 +694,26 @@ time.
 For example, suppose we have the following set of classes:
 
 ```php
-use SilverStripe\CMS\Model\SiteTree;
+namespace {
```

Comment:
They aren't pointless. They're required to pass linting, and they're also reflective of how the `Page` and `PageController` classes are declared in the recipe.

Reply:
Yeah, lots of people in bespoke say they just delete the redundant namespace since it indents everything.  We should update those files in the recipe

> They're required to pass linting

Then seems like we should make an exception in linting rather than enforcing an annoying standard.

Reply:
I don't think it's an annoying standard? It matches what our recipes have, and it is best practice to leave these in. If bespoke choose to remove them, that's up to them. I argue that it isn't redundant, it's explicit - and being explicit is good.

Reply:
Honestly, I've been ignoring those `namespace` statement for the last 5+ years and never bothered to check what their purpose was.

Are they there to get around the fact PSR-4 would want Page to be in a namespace, but Page needs to be in the root namespace?

Reply:
I don't know what they're there for in the recipe - but they're there. If we're removing them we should remove them from the recipe. If we're not removing them from the recipe we should keep them in the docs.

Currently they're in the recipe, so it makes sense to put them in the docs in this PR. Removing them should be a separate card, if that's what we decide we want to do.

Reply:
I'd say delete from recipe

Reply:
I tried to track down when we added those `namespace` statement. That's what I found https://github.com/silverstripe/recipe-cms/pull/11#issuecomment-378854003

Doesn't look like there was much thought that went into this.

We also have this old RFC which doesn't look like it ever got implemented. https://github.com/silverstripe/silverstripe-framework/issues/5844

In the interest of moving this along, I suggest we leave the `namespace` statement in the doc. For now, if someone cares deeply about removing the `namespace` statement, raise a separate card.